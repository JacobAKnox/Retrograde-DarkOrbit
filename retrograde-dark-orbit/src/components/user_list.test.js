import { render } from '@testing-library/react'
import UserList from './user_list'

test('renders on-screen', () => {
  const page = render(<UserList/>);
  expect(page).toMatchSnapshot();
});
