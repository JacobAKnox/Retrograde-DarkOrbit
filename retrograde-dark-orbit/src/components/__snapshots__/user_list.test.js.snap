// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserList Component renders correctly and matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex items-center justify-center bg-gray-700 min-h-[516px] rounded-xl mb-4"
  >
    <div
      class="user-list bg-gray-800 p-2 rounded-lg shadow-lg"
      style="min-height: 550px; min-width: 220px;"
    >
      <ul
        class="list-none space-y-1"
      />
    </div>
  </div>
</DocumentFragment>
`;
